src/resources/orders/shipments.ts

```diff
@@ -0,0 +1,253 @@
+import { defaults, get, startsWith } from 'lodash';
+
+import Shipment from '../../models/Shipment';
+import List from '../../models/List';
+import ApiException from '../../exceptions/ApiException';
+import OrdersBaseResource from './base';
+import { ICreateParams, IGetParams, IListParams, IUpdateParams } from '../../types/shipment/params';
+import { CreateCallback, GetCallback, ListCallback, UpdateCallback } from '../../types/shipment/callback';
+import Order from '../../models/Order';
+import Resource from '../../resource';
+
+/**
+ * The `order_shipments` resource
+ *
+ * @since 2.2.0
+ */
+export default class OrdersShipmentsResource extends OrdersBaseResource {
```

Comment:
What about `OrderShipmentsResource`?

Reply:
I've tried to make these consistent with the actual API endpoint:

```
/**
 * The `customers_mandates` resource
 *
 * @since 1.2.0
 */
export default class CustomersMandatesResource
```
but if that is an issue, we can change the names.